assignments/lambda-classes.js

```diff
@@ -1 +1,152 @@
 // CODE here for your Lambda Classes
+class Person {
+    constructor(testingPartOne){
+        this.name = testingPartOne.name;
+        this.age = testingPartOne.age;
+        this.location = testingPartOne.location;
+        this.gender = testingPartOne.gender;
+    }
+    speak(){
+        return `Hello my name is ${this.name}, I am from ${this.location}.`;
+    }
+}
+
+class Instructor extends Person{
+    constructor(testingPartTwo){
```

Comment:
It would be much better and easier to read, if you used more descriptive parameter names here instead of `testingPartTwo`. In this instance, something like `instructorProps` makes a bit more semantic sense.